lib/ansible/modules/crypto/openssh_cert.py

```diff
@@ -402,6 +414,12 @@ def _check_perms(module):
             file_args = module.load_file_common_arguments(module.params)
             return not module.set_fs_attributes_if_different(file_args, False)
 
+        def _check_serial_number():
+            if self.serial_number is None:
+                # the default serial number set by ssh-keygen is 0
+                return serial_number == '0'
```

Comment:
I would expect the serial number to be ignored in case `serial_number` is not specified in the module. @MarkusTeufelberger what do you think?

Reply:
Yeah, if I create an ssh-cert with number 123 manually and then execute this module without setting a serial number, I'd expect that it wouldn't care about the number - not start complaining that the number isn't 0.

Reply:
So let's do
```.py
if self.serial_number is None:
    return
```
instead :)

Reply:
or even just `if not self.serial_number:`

Reply:
That would ignore the explicit wish of the user to have serial number `0`.

Reply:
Freakin' truthyness...

Reply:
Fixed.